src/Application/DataValueTranslator.php

```diff
@@ -22,7 +22,8 @@ public function translate( DataValue $dataValue ): mixed {
 		}
 
 		if ( $dataValue instanceof TimeValue ) {
-			return ltrim( $dataValue->getTime(), '+-' );
+			// TODO: handle date precision explicitly.
+			return str_replace( '-00', '-01', ltrim( $dataValue->getTime(), '+-' ) );
```

Comment:
This replaces the "missing" date parts for year/month/day precisions. Other precisions are not supported.